src/simularium/VisData.ts

```diff
@@ -376,15 +381,29 @@ class VisData {
         }
     }
 
+    public setCacheDisabled(cacheDisabled: boolean): void {
+        this.disableCache = cacheDisabled;
+    }
+
     // Add parsed frames to the cache and save the timestamp of the first frame
     private addFramesToCache(frames: ParsedBundle): void {
+        if (this.disableCache) {
+            this.trimCacheHead();
```

Comment:
it seems like you could just return here?

Reply:
So the cache is pretty tightly integrated into the app/animate cycle and needs to have at least the current frame.

Reply:
if "disabled" means only cache one frame, then maybe all you need to do is:
```
this.frameDataCache = [frames.frameDataArray];
this.frameCache = [frames.parsedAgentDataArray];
```
instead of trimCacheHead.